Blahgger/Views/Account/Login.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@model Blahgger.Models.LoginUser
+
+@{
+    ViewBag.Title = "Login";
+}
+
+<h2>Login</h2>
+
+@using (Html.BeginForm()) 
+{
+    @Html.AntiForgeryToken()
```

Comment:
Don't forget to add the `ValidateAntiForgeryToken` attribute to the POST action method.